MarkdownKit/Sources/Common/MarkdownParser.swift

```diff
@@ -154,15 +151,16 @@ open class MarkdownParser {
   }
 
   fileprivate func updateDefaultElements() {
+    // Parsing order matters!
```

Comment:
😏

Reply:
Would be better if it didn't ^^ I'll improve the element styling if I have time